### DISCUSSION TITLE
How to show name

### DISCUSSION THREAD

**Discussion by webhostsg** [roflcoopter/viseron#866] [Q&A]
Hi,

How do I get the `latest_thumbnail` to show the recognised face name instead of the object name? 

![CleanShot 2024-12-23 at 23 15 49](https://github.com/user-attachments/assets/ebbd542c-db40-40f3-9c75-4ee1d83aa116)

**Comment by roflcoopter** [comment]
That is not possible sadly, since the thumbnail comes from the detected object and not the face recognition.

The v3 beta does save face recognition snapshots tho so the name could be added to those.
It would also be beneficial to save the latest face recognized to a new file `latest_face_recognition_snapshot` or something like that.

Would that solve your issue?

**Reply by webhostsg** [reply]
It would be great if there was a new entity for the latest face recognition snapshot.
